app/streams/publishing_api/content_item.rb

```diff
@@ -0,0 +1,45 @@
+class PublishingAPI::ContentItem
+  attr_reader :content_id
+  attr_reader :base_path
+  attr_reader :payload_version
+  attr_reader :locale
+  attr_reader :metadata
+  attr_reader :details
+  attr_reader :links
+
+  def self.parse_message(message_payload)
+    content_id = message_payload.fetch('content_id')
+    base_path = message_payload.fetch('base_path')
+    payload_version = message_payload.fetch('payload_version')
+    locale = message_payload['locale'] || 'en'
+    metadata = Item::Metadata::Parsers::Metadata.parse(message_payload)
```

Comment:
If we are getting the metadata at this point I would move that code inside the `streams` folder, so we don't couple the stream to `etl/item`

Reply:
Move the parser you mean? That sounds like a good idea, I think everything under `etl/item` and `etl/items` is actually only used by the `streams` code.

Reply:
I mean move only the `Metadata` parser. The other parsers like PDF counter... I would leave them in the `etl/item`...

Reply:
I'm going to do this in a separate PR because I need to refactor `MessageProcessor` more before this can be removed from the existing `Parser` class.